06-files-input-and-output/high_scores.py

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) 2016 Ami . All rights reserved
+
+import csv
+import os
+
+def parse_csv(file):
+    data = {}
+    print
+    with open(file, "r") as csv_file:
+        try:
+            csv_reader = csv.reader(csv_file)
+            for name, score in csv_reader:
+                score = int(score)
+                if name not in data:
```

Comment:
This `if`/`else` clause can be replace with a one liner:
`data[name] = max(data.get(name), score)`
where `get` will return None incase the name is not in the dictionary, and None is considered less than any number.